components/Page.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {injectGlobal} from 'styled-components';
+import Head from 'next/head';
+import Navigation from './Navigation';
+
+injectGlobal`
```

Comment:
I realized that these styles are loaded again for everytime Hot Module Reloading takes place... Apart from it being annoying it doesn't break anything and it shouldn't happen in production. Just FYI, so that you are not wondering in case it is happening to you too.